controllers/animation_recorder/animation_recorder.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+from controller import Supervisor
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--duration', type=float, default=10, help='Duration of the animation in seconds')
+    parser.add_argument('--output', default='../../animation/index.html', help='Path at which the animation will be saved')
+    args = parser.parse_args()
+
+    robot = Supervisor()
+    timestep = int(robot.getBasicTimeStep())
+    robot.animationStartRecording(args.output)
+
+    step_i = 0
+    n_steps = (1e3 * args.duration) / robot.getBasicTimeStep()
```

Comment:
1000 is more readable than 1e3 for neophytes.
```suggestion
    n_steps = (1000 * args.duration) / robot.getBasicTimeStep()
```